config/dca/cadesGPUCondo.cmake

```diff
@@ -110,7 +112,7 @@ string(CONCAT CTEST_BUILD_OPTIONS ${CTEST_BUILD_OPTIONS}
 # Setup a slurm job submission template
 # note that this is intentionally multiline
 #######################################################################
-set(PYCICLE_JOB_SCRIPT_TEMPLATE "#!/bin/bash
+set(PYCICLE_PBS_TEMPLATE "#!/bin/bash
```

Comment:
same here - I renamed it to PYCICLE_JOB_SCRIPT_TEMPLATE to make it easier to search/replace and switch between slurm/pbs /other